src/src_cpp/src_sequential/JsonWriter.cpp

```diff
@@ -134,6 +134,12 @@ void JsonWriter::write(int const & nbWeeks_p, BendersTrace const & bendersTrace_
     double gap_l = bendersTrace_p.back().get()->_bestub - bendersTrace_p.back().get()->_lb;
     _output["solution"]["gap"] = gap_l;
     _output["solution"]["optimality"] = (gap_l < 1e-03);
+    if (gap_l < 1e-03) {
+        _output["solution"]["problem_status"] = "OPTIMAL";
+    }
+    else {
+        _output["solution"]["problem_status"] = "ERROR";
+    }
```

Comment:
It looks like `_output["solution"]["problem_status"]` and `_output["solution"]["optimality"]` contain the same information. Consider removing one of them ?

Reply:
Yes, we could remove the boolean one, _output["solution"]["optimality"], as it can not be as precise as the string _output["solution"]["problem_status"] which can describe different error types or sub-optimality of a returned solution.